inference-engine/tests/ngraph_functions/include/ngraph_functions/builders.hpp

```diff
@@ -426,6 +426,12 @@ std::shared_ptr<ngraph::Node> makeRNN(const OutputVector& in,
                                       bool make_sequence = false,
                                       ngraph::op::RecurrentSequenceDirection direction = ngraph::op::RecurrentSequenceDirection::FORWARD);
 
+std::shared_ptr<ngraph::Node> makeGatherND(
+        const ngraph::Output<Node>& dataNode,
```

Comment:
may I ask you to make code alignment for this function definition like for other function definitions in this file?

Reply:
FIxed